pkg/provisioningv2/kubeconfig/manager.go

```diff
@@ -30,6 +30,7 @@ import (
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+	capi "sigs.k8s.io/cluster-api/api/v1beta1"
```

Comment:
FYI, we've been trying to minimize usage of the cluster-api packages to facilitate easier compatibility matrices.

That being said, it looks like the import of the APIs here are only to use the ClusterNameLabel constant which is itself a burden to maintain if we don't just import from CAPI, and we already reference this constant in other parts of the codebase numerously.